src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -361,7 +361,6 @@ enum AppStrings {
 		static let lowRiskExposureSubtitle = NSLocalizedString("ExposureDetection_LowRiskExposure_Subtitle", comment: "")
 		static let lowRiskExposureBody = NSLocalizedString("ExposureDetection_LowRiskExposure_Body", comment: "")
 
-		static let buttonEnable = NSLocalizedString("ExposureDetection_Button_Enable", comment: "")
```

Comment:
Is `ExposureDetection_Button_Enable` still used somewhere else or can it be removed from the strings file as well?

Reply:
I removed it from the strings file as well